sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,25 +2,29 @@
 
 ## 7.0.0b8 (Unreleased)
 
-**Breaking Changes**
-  - Renamed `AutoLockRenew` to `AutoLockRenewer`
-
 **New Features**
 
 * Added support for `timeout` parameter on the following operations:
   - `ServiceBusSender`: `send_messages`, `schedule_messages` and `cancel_scheduled_messages`
   - `ServiceBusReceiver`: `receive_deferred_messages` and `peek_messages`
   - `ServiceBusSession`: `get_state`, `set_state` and `renew_lock`
   - `ReceivedMessage`: `renew_lock`
+* `azure.servicebus.exceptions.ServiceBusError` now inherits from `azure.core.exceptions.AzureError`.
 
 **Breaking Changes**
 
+* Renamed `AutoLockRenew` to `AutoLockRenewer`.
+* The parameter `inner_exception` that `ServiceBusError.__init__` takes is now renamed to `error`.
+* Renamed `azure.servicebus.exceptions.MessageError` to `azure.servicebus.exceptions.ServiceBusMessageError`
+* Removed error `ServiceBusResourceNotFound` as `azure.core.exceptions.ResourceNotFoundError` is now raised when a Service Bus
+resource does not exist when using the `ServiceBusAdministrationClient`.
 * Message settlement methods (`complete`, `abandon`, `defer` and `dead_letter`)
 and methods that use amqp management link for request like `schedule_messages`, `received_deferred_messages`, etc.
 now raise more concrete exception other than `MessageSettleFailed` and `ServiceBusError`.
 * Exceptions `MessageSendFailed`, `MessageSettleFailed` and `MessageLockExpired`
  now inherit from `azure.servicebus.exceptions.MessageError`.
-* Removed Exception `ServiceBusResourceNotFound` as `azure.core.exceptions.ResourceNotFoundError` is now raised when a Service Bus resource does not exist.
+* Removed Exception `ServiceBusResourceNotFound` as `azure.core.exceptions.ResourceNotFoundError` is now raised when a
```

Comment:
This seems to be a duplicate of the entry above

Reply:
thanks for pointing out, removed